### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59459]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
When moving a message to another folder, you're unable to successfully move the message when using a keyboard only. Selecting a radio button using keyboard and tabbing to the Confirm button results in an error, while clicking on a radio button and clicking Confirm does not.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Log in with a test user and navigate to a message thread.
2. Press [tab] until keyboard focus moves to the Move button.
3. Press [enter] to open the "Move to" modal.
4. Press [tab] to move focus to the first radio button.
5. Press [spacebar] to select the first radio button, or use arrow keys to select one of the other radio buttons. Note that the radio button you select appears to be selected and the `va-radio-option` component has a `checked` attribute.
6. Press [tab] to move focus to the Confirm button.
7. Press [enter] to move the message.
8. Note that you receive an error message, "Please select a folder to move the message to."

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

### Platform Recommendation
I'm guessing whatever is validating the selection here is listening for a click event, rather than checking to see if one of the options is checked?

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by bryan-riley-va** [comment]
JIRA ticket created: https://jira.devops.va.gov/browse/MHV-45989

**Comment by AngelaFowler82** [comment]
Confirming that I can now move a message to another folder using the keyboard. Closing this as successfully resolved.
